runner/runners/invoke.go

```diff
@@ -171,9 +171,13 @@ func (inv *Invoker) run(cmd *runner.Command, id runner.RunID, abortCh chan struc
 	// Wait for process to complete (or cancel if we're told to)
 	select {
 	case <-abortCh:
+		stdout.Write([]byte(fmt.Sprintf("%s\n\nTask aborted: %v", marker, cmd.String())))
```

Comment:
This will not include the current contents of stdout/stderr from the running command will it?  If not, can we modify this (and timeout below) to include the current stdout/stderr?